client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -201,8 +199,10 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		case 'not-authorized':
```

Comment:
I was having a difficult time reproducing this scenario, it makes me wondering that maybe we want to move [this condition](https://github.com/Automattic/wp-calypso/blob/13407976668b2f293b4a4081f8b8bee9efe7514d/client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx#L176-L178) above the update plugin check since it relies on the `sourceSiteId ` anyway, what do you think?

Reply:
We both agreed that we would not move the condition here, so I'll resolve this conversation.